devices/tuya/_TZE200_pay2byax_door_sensor.json

```diff
@@ -0,0 +1,69 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": ["_TZE200_pay2byax", "_TZE200_kltffuzl", "_TZE200_fwoorn8y"],
+  "modelid": ["TS0601", "TS0601", "TS0601"],
+  "product": "ZG-102Z",
```

Comment:
I guess that should include the device type as well 🙂

Reply:
Renamed as _TZE200_pay2byax_openclose_sensor
All tuya stuff is using the manufacture name as file name https://github.com/dresden-elektronik/deconz-rest-plugin/tree/master/devices/tuya

>I guess that should include the device type as wel

You mean "product" ? I added some model